### PR TITLE
Initial code. Handle current TSS contract

### PR DESCRIPTION
**Pull request by ceigel**

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -0,0 +1,13 @@
+language: ruby
+rvm:
+  - 2.3.0
```
Comment (cs) [inline .travis.yml]: We could add a few more Ruby versions here. For instance, our core project is still running on `2.1.x`.

File: lib/tss/connector.rb [hunk 2]
```diff
@@ -0,0 +1,93 @@
...
+    end
+
+    def base_path
+      "/v1/#{oid}"
+    end
+
+    def handle_post_response(response, path)
+      case response.code
+      when 200, 201 then JSON.parse(response.body)
+      else fail "RSS: 'POST #{path}' failed with status = #{response.code}."
```
Comment (cs) [inline lib/tss/connector.rb]: TSS

File: lib/tss/connector.rb [hunk 3]
```diff
@@ -0,0 +1,93 @@
...
+    attr_reader :oid
+    def initialize(organization_id)
+      @oid = organization_id
+    end
+
+    # @param [String] - organization_id
+    # @raise [RuntimeError] TSS request failed
+    def organization
+      path = "/v1/organizations/#{oid}"
+      p path
```
Comment (cs) [inline lib/tss/connector.rb]: debug, debug, debug :)

File: tss-connector.gemspec [hunk 4]
```diff
@@ -0,0 +1,33 @@
...
+  spec.files = Dir["{lib,spec}/**/*.rb"] + ['README.md', 'Rakefile']
+  spec.test_files = Dir["spec/**/*.rb"]
+
+  spec.bindir        = 'exe'
+  spec.executables   = spec.files.grep(%r{^exe/}) { |f| File.basename(f) }
+  spec.require_paths = ['lib']
+
+  spec.add_dependency 'httparty', '~> 0.10.2'
+  spec.add_dependency 'activesupport', '~> 3.2'
+  spec.add_dependency 'tzinfo', '~> 0.3'
```
Comment (cs) [inline tss-connector.gemspec]: `tzinfo` redundant...

File: Gemfile [hunk 5]
```diff
@@ -0,0 +1,4 @@
+source 'https://rubygems.org'
+
+# Specify your gem's dependencies in rss-connector.gemspec
```
Comment (cs) [inline Gemfile]: tss

File: lib/tss/connector.rb [hunk 6]
```diff
@@ -0,0 +1,93 @@
...
+    # @raise [RuntimeError] TSS request failed
+    def transactions
+      path = "#{base_path}/transactions"
+      response = self.class.authenticated_get(path)
+
+      handle_get_response(response, path, 'transactions')
+    end
+
+    # @raise [RuntimeError] TSS request failed
+    def transaction(transaction_id)
```
Comment (cs) [inline lib/tss/connector.rb]: I would reconsider the naming of methods like this one... maybe rename `#transaction` to `#get_transaction`, `#organization` to `#get_organization` and so on.

File: lib/tss/connector.rb (hunk 3 above)
Comment (ceigel) [inline lib/tss/connector.rb]: :)

**Comment by cs** [comment]
:+1:

**Pull request merged**